mcfly/find_architecture.py

```diff
@@ -36,12 +36,12 @@
 from tensorflow.keras import metrics
 from tensorflow.keras.callbacks import EarlyStopping
 
-from . import modelgen
+import modelgen
 
 
 def train_models_on_samples(X_train, y_train, X_val, y_val, models,
-                            nr_epochs=5, subset_size=100, verbose=True, outputfile=None,
-                            model_path=None, early_stopping=False,
+                            nr_epochs=5, subset_size=None, verbose=True, outputfile=None,
```

Comment:
This is not a better default than 100. I suggest all subset_size defaults back to 100.

Reply:
I set it back to 100 and added print statement to show when only a subset of the data is used for training.